### PR TITLE
Use github issue forms

### PR DESCRIPTION
**Pull request by floryst**
### Context
Switch to using github issue forms, which gives users a better way to report issues and feature requests.

Please look over the contents and suggest wording, additional fields (try to minimize # of fields, especially required ones), etc. @aylward @tbirdso @PaulHax 

### Results
We get nicer issue input forms.

### Changes
- [x] Update github issue templates

### PR and Code Checklist

- [x] [semantic-release](https://github.com/semantic-release/semantic-release) commit messages
- [x] Run `npm run reformat` to have correctly formatted code

### Testing
I haven't figured out how to test this without merging and seeing the results.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    attributes:
+      label: Bug description
+      description: A description of the encountered bug.
+    validations:
+      required: true
+  - type: textarea
+    id: repro
+    attributes:
+      label: Steps to reproduce
+      description: A list of steps and/or code snippets required to reproduce the behavior. A live example (e.g. codesandbox, codepen) is preferred.
```
Comment (finetjul) [inline .github/ISSUE_TEMPLATE/bug_report.yml:28]: A live example (e.g. VTK.js examples, codesandbox, codepen...)

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+assignees:
+body:
+  - type: markdown
+    attributes:
+      value: |
+        👋 Hello, and thank you for starting this contribution!
+
+        ❓ If you are trying to submit a question for general support, please reach out on our discourse forum instead: https://discourse.vtk.org/c/web/9
+
+        🐞 If you do need to report a bug, please follow the template below to increase the chances of your report being looked at.
```
Comment (finetjul) [inline .github/ISSUE_TEMPLATE/bug_report.yml:14]: follow the template below to increase -> please complete the template below as much as possible to increase

**Review by finetjul — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (PaulHax) [reply .github/ISSUE_TEMPLATE/bug_report.yml:28]: Yes!  Is there a template for one of the live example sites?  Based this one off of a @jourdain one a while back: https://stackblitz.com/edit/js-bqmqyf

**Review by tbirdso — COMMENTED** [review]
Neat!

When I click on the top-right dropdown for any of the `.yml` templates and click "View File" I see the following warnings. Not sure how critical these are.


> There are some problems with this template

> title must be of type String and cannot be empty. [Learn more](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/common-validation-errors-when-creating-issue-forms#key-must-be-a-string).

> assignees must be of type String or Array. [Learn more](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/common-validation-errors-when-creating-issue-forms#key-must-be-a-string).

Reference: https://github.com/Kitware/vtk-js/blob/3579677fb31e49c87d2dcfa2883c6d648a5159cc/.github/ISSUE_TEMPLATE/bug_report.yml

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (finetjul) [reply .github/ISSUE_TEMPLATE/bug_report.yml:28]: There is one listed in the performance template:
https://codepen.io/jourdain/pen/RQZWYa

**Comment by floryst** [comment]
Those errors don't seem that critical, but I'll address them anyways.

**Review by jourdain — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  - type: markdown
+    attributes:
+      value: |
+        👋 Hello, and thank you for starting this contribution!
+
+        ❓ If you are trying to submit a question for general support, please reach out on our discourse forum instead: https://discourse.vtk.org/c/web/9
+
+        💡 If you do want to suggest a change or request a feature, please follow the template below to increase the chances of your report being looked at.
+
+        🧱 If adequate, assign one of the `Module: XXX` labels to your issue when creating it.
```
Comment (aylward) [inline .github/ISSUE_TEMPLATE/feature_request.yml:16]: If adequate -> When applicable, please

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  - type: textarea
+    id: description
+    attributes:
+      label: Needs
+      description: A clear and concise description of what need or problem you're trying to address.
+    validations:
+      required: true
+  - type: textarea
+    id: feature
```
Comment (aylward) [inline .github/ISSUE_TEMPLATE/feature_request.yml:25]: Not a big fan of the categories "Needs" and "Feature Description" when requesting changes or new features.  Perhaps:
"Request" (description) and "Motivation"
"Request" (description) and "Justification"
"Request" (description) and Reason"

File: .github/ISSUE_TEMPLATE/performance_issue.yml [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+assignees:
+body:
+  - type: markdown
+    attributes:
+      value: |
+        👋 Hello, and thank you for starting this contribution!
+
+        ❓ If you are trying to submit a question for general support, please reach out on our discourse forum instead: https://discourse.vtk.org/c/web/9
+
+        🐞 If you do need to report a bug, please follow the template below to increase the chances of your report being looked at.
```
Comment (aylward) [inline .github/ISSUE_TEMPLATE/performance_issue.yml:14]: This should be removed?    The template is for Performance Issues, not bugs, correct?

File: .github/ISSUE_TEMPLATE/performance_issue.yml [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+        ⚡️ If you do need to report some performance issue, please follow the template below to increase the chances of your report being looked at.
```
Comment (aylward) [inline .github/ISSUE_TEMPLATE/performance_issue.yml:16]: some -> a

to increase the chances of your report being looked at -> to help us understand and address your request.

File: .github/ISSUE_TEMPLATE/performance_issue.yml [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+        🧱 If adequate, assign one of the `Module: XXX` labels to your issue when creating it.
```
Comment (aylward) [inline .github/ISSUE_TEMPLATE/performance_issue.yml:18]: If adequate -> When appropriate

File: .github/ISSUE_TEMPLATE/performance_issue.yml [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+          - profiling data
+          - videos (drag and drop)
+          - live examples (ex: https://codepen.io/jourdain/pen/RQZWYa)
+    validations:
+      required: true
+  - type: textarea
+    id: environment
+    attributes:
+      label: Environment
+      description: A list of affected version numbers and platforms.
```
Comment (aylward) [inline .github/ISSUE_TEMPLATE/performance_issue.yml:48]: affected version numbers and platforms -> relevant platforms and their version numbers

**Review by aylward — CHANGES_REQUESTED** [review]
LGTM, but I think some of the wording needs to be cleaned a bit

**Comment by floryst** [comment]
This is updated and ready for another pass. @aylward

**Comment by floryst** [comment]
Ping @aylward, since you have a "changes requested" flag on this PR.

**Review by aylward — APPROVED** [review]
Nice!

**Comment by daker** [comment]
@floryst i only see those two entries, i think it's due to the errors in [feature_request.yml](https://github.com/Kitware/vtk-js/blob/master/.github/ISSUE_TEMPLATE/feature_request.yml) and [bug_report.yml](https://github.com/Kitware/vtk-js/blob/master/.github/ISSUE_TEMPLATE/bug_report.yml)

image

**Pull request merged**